elsa-typed-builtin.el

```diff
@@ -283,9 +279,9 @@
 (put 'point 'elsa-type (elsa-make-type Int))
 (put 'point-marker 'elsa-type (elsa-make-type Marker))
 (put 'goto-char 'elsa-type (elsa-make-type Int | Marker -> Int))
-;; (put 'region-beginning 'elsa-type (elsa-make-type))
-;; (put 'region-end 'elsa-type (elsa-make-type))
-;; (put 'mark-marker 'elsa-type (elsa-make-type))
+(put 'region-beginning 'elsa-type (elsa-make-type Int))
```

Comment:
What happens if there is no mark? Can't it return nil or error?

Reply:
There isn't but will be: #2

Reply:
I don't have it figured out yet, but probably it will be a property on the symbol similar to the type annotations.  A list of signal types would probably be sufficient data.

Reply:
Yea something like that.  But there might be more info necessary, I don't know before I try to implement the checks for it.